app/views/request-an-account.html

```diff
@@ -0,0 +1,35 @@
+
+{% extends "_templates/_page.html" %}
+{% set backLink = '/start-page' %}
+{% set hideNav = true %}
+{% set hidePrimaryNav = true %}
+
+
+{% set pageHeading = 'Request an account' %}
+
+{% block content %}
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds-from-desktop">
+    <h1 class="govuk-heading-l govuk-!-margin-bottom-9">{{pageHeading}}</h1>
+
+    <p class="govuk-body">To get set up on Register trainee teachers (Register) you need a DfE Sign-in account.</p>
+    <p class="govuk-body">Get a <a href="https://services.signin.education.gov.uk/" class="govuk-link">DfE Sign-in account</a> if you do not already have one.</p>
+    <p class="govuk-body">Then request an account for Register trainee teachers by sending an email to <a href="mailto:becomingateacher@digital.education.gov.uk" class="govuk-link">becomingateacher@digital.education.gov.uk</a></p>
+    <p class="govuk-body">The email address associated with your DfE Sign-in and Register account must:</p>
+    <ul class="govuk-list govuk-list--bullet">
+      <li>be a named email address, for example jane.doe@organisation.com</li>
```

Comment:
```suggestion
      <li>be a named email address, for example jane.doe@example.com</li>
```

Doesn't look like www.organisation.com exists, but I think it _could_ - whereas `example.com` is meant for this sort of usage.